index.html

```diff
@@ -0,0 +1,260 @@
+        <div class="container">
+            <header>
+                <nav>
+                    <ul>
+                        <li id="my-logo-nav-item"><a href="#logo">My Logo</a></li>
+                        <li class="left-nav-items">Portfolio</li>
+                        <li class="left-nav-items">about</li>
+                        <li class="left-nav-items">contact</li>
+                        <li id="hamburger-nav-item">
+                            <img src="images/hamburger-icon.png" alt="hamburger
+                                icon">
+                        </li>
+                    </ul>
+                </nav>
+            </header>
+
+            <section id="hero-section">
+                <div id="hero-area-content">
+                    <h1>
+                        I’m Dyary<br>
+                        Glad to see you!
+                    </h1>
+                    <p>I’m a software developer! I can help you build a
+                        product
+                        ,
+                        feature or website Look through some of my work and
+                        experience!
+                        If you like what you see and have a project you need
+                        coded,
+                        don’t hestiate to contact me.
+                    </p>
+                    <label class="lets-connect-label">LETS CONNECT</label>
+                    <ul>
+                        <li><span><a><img src="images/twitter-icon.svg"
+                                        alt="Twitter
+                                        Icon"></a></span></li>
+                        <li><span><a><img
+                                        src="images/linked-in-icon.svg"
+                                        alt="LinkedIn
+                                        Icon"></a></span></li>
+                        <li><span><a><img src="images/medium-icon.svg"
+                                        alt="Medium
+                                        Icon"></a></span></li>
+                        <li><span><a><img src="images/github-icon.svg"
+                                        alt="Github
+                                        Icon"></a></span></li>
+                        <li><span><a><img src="images/angellist-icon.svg"
+                                        alt="Angellist
+                                        Icon"></a></span></li>
+                    </ul>
+                </div>
+            </section>
+        </div>
+
+
+        <section id="work-area-grid">
+
+            <section class="work-card first-card flex">
+                <img class="work-card-desktop-image"
+                    src="images/first-work-card.png" alt="Tonic work card">
+                <img class="work-card-mobile-image"
+                    src="images/first-work-card-mobile.png" alt="Tonic work
+                    card">
+                <div class="work-card-content">
+                    <h1>Tonic</h1>
```

Comment:
could you use a` <label>` tag with a large font instead of `<h1> `here. it would be more semantically appropriate. it's best to use headings for top-level text like the title of the page or their subtitles.